### DISCUSSION TITLE
How to specify the default completion style in `vertico-multiform-commands`?

### DISCUSSION THREAD

**Discussion by ijqq** [minad/vertico#547] [Q&A] [answered]
I want to use the flat style by default, but specify a few commands which should use the default vertical completion style instead.  In the snippet shown below, using `vertical` gives an error that the symbol definition is void, while other values such as `buffer`, `grid`, and `reverse` all work.

```lisp
(use-package vertico
  :ensure t
  :init
  (vertico-mode)
  (vertico-multiform-mode)
  :config
  (setq vertico-multiform-commands
	'((switch-to-buffer vertical)
	  (t flat))))
```

**Comment by minad** [comment, marked as answer]
It should work if you simply specify nothing, i.e., (switch-to-buffer), since vertical is the default.

**Reply by ijqq** [reply]
That works. Thank you very much.
